src/battle-dex.ts

```diff
@@ -215,6 +215,8 @@ const Dex = new class implements ModdedDex {
 	readonly statNames: ReadonlyArray<StatName> = ['hp', 'atk', 'def', 'spa', 'spd', 'spe'];
 	readonly statNamesExceptHP: ReadonlyArray<StatNameExceptHP> = ['atk', 'def', 'spa', 'spd', 'spe'];
 
+	pokeballs: {[k: string]: Item} = null!;
```

Comment:
I would make this a `string[] | null` or `ID[] | null`

Reply:
I guess `string[] | null` because we need the `.name`, and can get `.id` from `.name`